message_definitions/v1.0/common.xml

```diff
@@ -373,6 +373,23 @@
         <description>Velocity limiting active to prevent breach</description>
       </entry>
     </enum>
+    <enum name="FENCE_TYPE">
+      <entry value="0" name="FENCE_TYPE_ALL">
```

Comment:
can't have a 0 value in a bitmask

Reply:
Actually not true - there are lots currently in the same file